universal-login-sdk/src/core/utils/validation.ts

```diff
@@ -2,5 +2,5 @@ import {UnsignedMessage, ensure, GAS_BASE} from '@unilogin/commons';
 import {InsufficientGas} from './errors';
 
 export const ensureSufficientGas = (unsingedMessage: UnsignedMessage) => {
-  ensure(GAS_BASE < unsingedMessage.safeTxGas, InsufficientGas, `gasLimit should be greater than ${GAS_BASE}`);
+  ensure(GAS_BASE < unsingedMessage.safeTxGas, InsufficientGas, `Got safeTxGas ${unsingedMessage.safeTxGas} but should greater than ${GAS_BASE}`);
```

Comment:
Update signature of InsufficientGas to pass into the error just 2 parameters (values of safeTxGas)